src/dev/arm/smmu_v3_slaveifc.hh

```diff
@@ -72,7 +72,7 @@ class SMMUv3SlaveInterface : public ClockedObject
     const Cycles microTLBLat;
     const Cycles mainTLBLat;
 
-    SMMUSlavePort *slavePort;
+    SMMUSlavePort *responsePort;
```

Comment:
Similar to my comment above... I noticed you didn't change the ones below. Maybe this is a case where we should just ask :).